### PR TITLE
Waterfall charts doc

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #504.

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/waterfall.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+# Waterfall charts
+
+Waterfall charts can display a running total as values are added or subtracted. They are used to
+rapidly understand how an initial value is affected by a series of positive or negative changes.
+
+A typical use of waterfall charts shows how a net value is reached through gains and losses
+over time.
+
+To create a waterfall chart in Taipy, you must set the [*type*](../chart.md#p-type) property
+of the chart control "waterfall".
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/waterfall.md]: to "waterfall" ?

File: docs/manuals/gui/viselements/charts/waterfall.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+want to display the running total: the cumulative difference between the actual score and the par
+for each hole. All those differences are stored in another array that we store in the *data*
+dictionary:
+
+```py
+data["Diff"] = [data["Score"][i]-data["Par"][i] for i in range(0,n_holes)] + [None]
+```
+
+For each hole, we compute the difference between the actual score and the par (a value of -2,
+for example, indicates ). We then set the entire array of differences, with an additional None
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/waterfall.md]: indicates what ?

**Review by FredLL-Avaiga — APPROVED** [review]
👍

File: docs/manuals/gui/viselements/charts/waterfall.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/waterfall.md]: Good spot!

File: docs/manuals/gui/viselements/charts/waterfall.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/waterfall.md]: Who knows.

**Pull request merged**